macOS/DuckDuckGo/NetworkProtection/AppAndExtensionAndAgentTargets/UserDefaults+NetworkProtectionShared.swift

```diff
@@ -68,10 +69,12 @@ extension NetworkProtectionUI.OnboardingStatus {
     /// - For DeveloperID builds the default is asking the user to allow the System Extension.
     ///
     public static let `default`: OnboardingStatus = {
-#if NETP_SYSTEM_EXTENSION
-        .isOnboarding(step: .userNeedsToAllowExtension)
-#else
-        .isOnboarding(step: .userNeedsToAllowVPNConfiguration)
-#endif
+        let vpnAppState = VPNAppState(defaults: .netP)
+
+        if vpnAppState.isUsingSystemExtension {
+            return .isOnboarding(step: .userNeedsToAllowExtension)
+        } else {
+            return .isOnboarding(step: .userNeedsToAllowVPNConfiguration)
+        }
```

Comment:
`NETP_SYSTEM_EXTENSION` went away for this target in favor of `isUsingSystemExtension`, since this is now a dynamic setting.